bears/uploadTool.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+import glob
+import os
+import re
+import shutil
+import subprocess
+from coalib.collecting.Importers import iimport_objects
+from string import Template
+
+
+def touch(file_name):
+    """
+    Creates an empty file.
+
+    :param file_name: Name of the file.
+    """
+    if not os.path.exists(file_name):
+        open(file_name, 'w').close()
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Choose to upload'
+                                     'the bears or not')
+    parser.add_argument('-u', '--upload', help='Upload the bears',
+                        action='store_true')
+    args = parser.parse_args()
+
+    os.makedirs('upload', exist_ok=True)
+
+    bears_full_filenames = glob.glob('**/*Bear.py')
+
+    for bear_file_name in bears_full_filenames:
+        file_name = bear_file_name
+        bear_name = re.match(r'.+/(.*)\.py', bear_file_name).group(1)
+        bear_object = next(iimport_objects(file_name, attributes='kind',
+                                           local=True))
+        os.makedirs(os.path.join('upload', bear_name), exist_ok=True)
+        os.makedirs(os.path.join('upload', bear_name, bear_name), exist_ok=True)
+        touch(os.path.join('upload', bear_name, bear_name,
```

Comment:
you should put `os.path.join('upload', bear_name, bear_name)` into a variable :)